src/apps/front-office/shop/pages/ComparePage/ComparePage.tsx

```diff
@@ -0,0 +1,101 @@
+import { trans } from "@mongez/localization";
+import Helmet from "@mongez/react-helmet";
+import { cn } from "@utils";
+import HorizonalTable from "src/apps/front-office/design-system/components/HorizonalTable";
+import Rating from "src/apps/front-office/design-system/components/Rating";
+import Breadcrumb from "src/apps/front-office/design-system/layouts/Breadcrumb";
+import URLS from "src/apps/front-office/utils/urls";
+import compareProducts, { StackStatus } from "../mocks/compare-products";
+
+export default function ComparePage() {
+  return (
+    <>
+      <Helmet title={trans("compare.title")} />
+      <main className="w-full text-secondary">
+        <div className="breadcrump relative">
+          <div className="absolute w-full border-b-2 border-b-gray-200 top-full"></div>
+          <div className="nav-links w-full">
+            <Breadcrumb
+              navItems={[
+                { name: "Pages" },
+                { name: "Compare Products", url: URLS.compare },
+              ]}
+              attributes={{
+                className: "justify-end items-end py-4 my-0",
+              }}
+            />
+          </div>
+        </div>
+
+        <section className="my-10">
+          <h2 className="text-4xl font-bold my-4">Compare Products</h2>
+          <p className="text-muted-foreground font-bold">
+            There are <span className="text-primary">3</span> products to
+            compare
+          </p>
+
+          <HorizonalTable
+            columnsAside={[
+              { key: "image", title: "Preview" },
+              { key: "name", title: "Name" },
+              { key: "price", title: "Price" },
+              { key: "rating", title: "Rating" },
+              { key: "description", title: "Description" },
+              { key: "stackStatus", title: "Stack Status" },
+              { key: "weight", title: "Weight" },
+              { key: "dimensions", title: "Dimensions" },
+            ]}
+            data={compareProducts}
+            render={{
```

Comment:
Making a generic component like `HorizonalTable` will go against specifying render items like `name`, `image` and `price`, otherwise what are the other uses for that component?

Reply:
in new PR: I put this component only in shop module instead of main design-system.